app/workers/import_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ImportWorker
+  include Sidekiq::Worker
+
+  def perform(device, json_file)
```

Comment:
@t-harps Not sure what you mean. When I put dot sign 30_000 times in my console it happens immediately.